### PR TITLE
Guard minor tweaks

### PR DESCRIPTION
**Pull request by weaverryan**
| Q | A |
| --- | --- |
| Bug fix? | yes |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |

Various completely minor things, most from suggestions on #14673

### CODE REVIEW

File: src/Symfony/Component/Security/Guard/Tests/Firewall/GuardAuthenticationListenerTest.php [hunk 1]
```diff
@@ -218,5 +218,6 @@ protected function tearDown()
         $this->event = null;
         $this->logger = null;
         $this->request = null;
+        $this->rememberMeServices = null;
```
Comment (WouterJ) [inline src/Symfony/Component/Security/Guard/Tests/Firewall/GuardAuthenticationListenerTest.php]: Can't we simply combine this by changing all `=`s?

**Comment by WouterJ** [comment]
Status: Reviewed

File: src/Symfony/Component/Security/Guard/Tests/Firewall/GuardAuthenticationListenerTest.php (hunk 1 above)
Comment (stof) [inline src/Symfony/Component/Security/Guard/Tests/Firewall/GuardAuthenticationListenerTest.php]: I think this is more readable though.

**Comment by fabpot** [comment]
Thank you @weaverryan.

**Pull request merged**